constructors.js

```diff
@@ -161,7 +187,15 @@ console.assert(safe.unlock(rightKey) === sensitive, 'valid keys should open the
 
 // -- ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ - Your Answer - ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ ↓ -- //
 
-
+function Validator() {
+  this.email = function(email) {
+    if (email.indexOf('@') >= 0) {
```

Comment:
Look for more than just the index of `@`. At least make this check somewhat complex like `@` AND a `.`